wcfsetup/install/files/acp/templates/__optionRewriteRules.tpl

```diff
@@ -0,0 +1,14 @@
+{if $category->categoryName === 'general'}
+	<div id="dialogRewriteRules" style="display: none" />
```

Comment:
The JavaScript can generate the element itself (see https://docs.woltlab.com/javascript_new-api_dialogs.html)